sentry-sidekiq/lib/sentry/sidekiq-scheduler/scheduler.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+return unless defined?(::SidekiqScheduler::Scheduler)
```

Comment:
I think we should try require [sidekiq-scheduler](https://github.com/sidekiq-scheduler/sidekiq-scheduler) here? Otherwise, it's not guaranteed that it will be required before `sentry-sidekiq` and then this will return even though the gem is present in Gemfile.

Reply:
Yes so I also added a comment there. I think something like this should be enough?

```rb
begin
  require "sidekiq-scheduler"
rescue LoadError
  return
end
```

Reply:
@st0012 @sl0thentr0py I'm happy to clean up both, sidekiq-cron in a separate PR to keep things clear.